tests/tokens/themes.spec.js

```diff
@@ -0,0 +1,125 @@
+const { mainTest } = require('../../fixtures');
+const { MainPage } = require('../../pages/workspace/main-page');
+const { expect, test } = require('@playwright/test');
+const { random } = require('../../helpers/string-generator');
+const { TeamPage } = require('../../pages/dashboard/team-page');
+const { DashboardPage } = require('../../pages/dashboard/dashboard-page');
+const { updateTestResults } = require('./../../helpers/saveTestResults.js');
+const { qase } = require('playwright-qase-reporter/dist/playwright');
+const { TokensPanelPage } = require('../../pages/workspace/tokens-panel-page');
+const { DesignPanelPage } = require('../../pages/workspace/design-panel-page');
+
+let mainPage, teamPage, dashboardPage, tokensPage, designPanelPage;
+
+const teamName = random().concat('autotest');
+
+test.beforeEach(async ({ page, browserName }) => {
+  teamPage = new TeamPage(page);
+  dashboardPage = new DashboardPage(page);
+  mainPage = new MainPage(page);
+  tokensPage = new TokensPanelPage(page);
+  designPanelPage = new DesignPanelPage(page);
+  await teamPage.createTeam(teamName);
+  await teamPage.isTeamSelected(teamName);
+  await dashboardPage.createFileViaPlaceholder();
+  await mainPage.isMainPageLoaded();
+  await mainPage.clickMoveButton();
+});
+
+test.afterEach(async ({ page }, testInfo) => {
+  await mainPage.backToDashboardFromFileEditor();
+  await teamPage.deleteTeam(teamName);
+  await updateTestResults(testInfo.status, testInfo.retry);
+});
+
+mainTest(qase(2167, 'Create theme via "create one" link'), async () => {
+  await tokensPage.clickTokensTab();
+  await tokensPage.createThemeViaLink('Desktop');
+  await tokensPage.checkSelectedTheme('No theme active');
+});
+
+mainTest(qase(2206, 'Enable themes in different groups'), async () => {
```

Comment:
We have an update of this [test case](https://app.qase.io/project/PENPOT?case=2188) (pre-condition of [#2206](https://app.qase.io/project/PENPOT?case=2206)) related to this [enhancement](https://tree.taiga.io/project/penpot/issue/10746).
Could you please review it and update the test according to the changes?